stock_lot_internal_code/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Stock Lot Internal Code",
+    "version": "16.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "stock",
+    "license": "LGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
```
Let's apply AGPL as it's not really expected to be a dependency of another module.